src/components/Typography/BodyX/BodyX.stories.tsx

```diff
@@ -73,14 +73,14 @@ export const All = {
     children: { table: { disable: true } },
   },
   render: (args: object) => (
-    <>
+    <div style={{ display: 'flex', flexDirection: 'column', gap: 16 }}>
```

Comment:
I would either extract a common style 
```
{ display: 'flex', flexDirection: 'column', gap: 16 }
```
or a common decorator
```
({children}) => (
    <div style={{ display: 'flex', flexDirection: 'column', gap: 16 }}>
    {children}
    </div>
    ```
    to be put in mocks file and reused in different stories. Same applies also for HX

Reply:
I extracted the `displayAll` style ✅